genai-perf/README.md

```diff
@@ -634,9 +637,11 @@ The concurrency value to benchmark. (default: `None`)
 ##### `--measurement-interval <int>`
 ##### `-p <int>`
 
-The time interval used for each measurement in milliseconds. Perf Analyzer
-will sample a time interval specified and take measurement over the requests
-completed within that time interval. (default: `10000`)
+The time interval used for each measurement in milliseconds.
+Perf Analyzer will sample a time interval specified and take
+measurement over the requests completed within that time interval.
+When using the default stability percentage, GenAI-Perf will benchmark
+for 3*(measurement_interval) milliseconds.
```

Comment:
Long-term, we may want to deprecate this arg in favor of something clearer, where GenAI-Perf natively converts to measurement interval e.g. --benchmarking-duration 1000 gets converted to --measurement-interval 333 under the hood. To avoid any kind of confusion with the PA command printed or prior behavior, I'm leaving this as is for now.